### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server Application Pools Edit_Screen Reader: After clicking on create application pool drop down Narrator is not reading whether it is expanded/collapsed

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#173]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)
Screen Reader
MAS ID Violated: MAS 1.3.1

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Switch on the Narrator
4.Tab till Microsoft IIS on top navigation bar and press enter
5.tab till web server and press enter
6.Tab till Application pool and press enter
7.Tab till create application pool drop down and press enter
8.Check whether Narrator is reading expanded/collapsed or not after clicking on it

Actual Result:
After clicking on create application pool drop down Narrator is not reading whether it is expanded/collapsed

Expected Result:
After click on drop down it should read expanded

User Impact:
Screen Reader users will not able understand whether the drop down expanded/collapsed

Mas Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D

Note: Same issue is  observed in most of the pages in the application

Reference Video:
[Bug_173.pptx](https://github.com/Microsoft/IIS.WebManager/files/2332096/Bug_173.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This will be fixed when we use WAC SDK controls.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Edge; #Narrator

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
